src/System.Reflection.Metadata/tests/Utilities/OrderByTests.cs

```diff
@@ -0,0 +1,32 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Reflection.Internal;
+using System.Text;
+using Xunit;
+
+namespace System.Reflection.Metadata.Tests
+{
+    public class OrderByTests
+    {
+        [Fact]
+        public void ElementsAllSameKey()
+        {
+            List<int> source = new List<int>(new int[] { 9, 9, 9, 9, 9, 9 });
+            int[] expected = { 9, 9, 9, 9, 9, 9 };
```

Comment:
This would pass irrespective of stability, right? Wouldn't it be better to have the basic test unrelated to stability at least prove that it's sorting?